bigchaindb/models.py

```diff
@@ -158,7 +158,7 @@ def from_db(cls, bigchain, tx_dict_list):
 
 # TODO: Remove node_pubkey as part of cleanup II
 class Block(object):
-    """Bundle a list of Transactions in a Block. Nodes vote on its validity.
```

Comment:
This block was is deprecated in favour of [this](https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/tendermint/lib.py#L374)